### DISCUSSION TITLE
Upcoming Rewrite / Version 14 unsupported

### DISCUSSION THREAD

**Discussion by kaaax0815** [wg-easy/wg-easy#1482] [Announcements]
We are excited to announce that we are in the process of rewriting **wg-easy** to make it even more user-friendly and accessible! 🎉

For those interested in tracking our progress, please check out the PR #1333. Any assistance is greatly appreciated.

Due to the time-consuming nature of this rewrite, we will not be able to address issues and bugs in version 14 thoroughly. However, major bugs will still be fixed.

Please note that new features built for the `master` branch will not be accepted, as the next major release, v15.0.0, will be this comprehensive rewrite (`feat-ui`)

### Exciting New Features in v15.0.0:
- UI Onboarding & Password management
- IPv6 Support
- CIDR Support
- Per Client Configuration
- Full UI Configuration (No more environment variables!)
- Documentation, Semver and much more

Read https://github.com/wg-easy/wg-easy/discussions/1179 to see whats planned for the future

A huge thanks to @tetuaoro for implementing major parts of the UI and backend.

Stay tuned for more updates!

If you have any questions or concerns, feel free to comment.

Thank you for your continued support and patience. We look forward to bringing you an even better wg-easy experience!

**Comment by lj249911023** [comment]
Can multiple servers be set up? (Multiple WireGuard networks.) And set the server to be able to access the client's local area network (2 clients can access each other's local area network).

**Reply by kaaax0815** [reply]
Not currently. Happy to review a PR after Version 15 is released

**Comment by PaulTankerfahrer** [comment]
Hey, love your work, any idea when v15 drops? Thanks and keep up the great work!

**Comment by racazahassem09191990** [comment]
I won't use it yet, I'll just wait. I'm excited about the upcoming version.

**Comment by kaaax0815** [comment]
We are thrilled to announce that the first beta of the new wg-easy rewrite will be released this or next week! 🚀

Thank you all for your patience and support. We can't wait for you to try out the new features and improvements. Stay tuned for more updates!

**Comment by issafram** [comment]
Regarding this bullet point:
> Full UI Configuration (No more environment variables!)

Does that include these variables that I use in docker compose?  They get me up and running very quickly.  Especially when recreating the container.
```
      - WG_HOST=wg.mydomain.com
      - WG_DEFAULT_ADDRESS=10.10.0.x
      - WG_DEFAULT_DNS=10.9.1.100
      - WG_DEVICE=eth0
```

**Reply by kaaax0815** [reply]
These env vars will still exist: https://wg-easy.github.io/wg-easy/Pre-release/advanced/config/optional-config/

If you want to configure some parts automatically when starting you can use this: https://wg-easy.github.io/wg-easy/Pre-release/advanced/config/unattended-setup/

**Comment by kaaax0815** [comment]
We’re excited to announce that wg-easy v15 is now officially released! 🚀

Thank you all for your patience and support throughout the development process. This update brings a host of new features and improvements—we can’t wait for you to experience it. Stay tuned for more updates!

**Reply by quyvsquy** [reply]
I apologize, but I attempted to use v15, however I prefer using v14 as it's simpler for me to edit the configuration through the wg0.json file. Using SQLite is more complex for my needs.

**Reply by kaaax0815** [reply]
Is there any reason for editing the configuration directly? At this point why not use WireGuard directly?
